data/styles/clear/include/Basemap.mapcss

```diff
@@ -670,34 +672,48 @@ area|z18-[building=train_station],
 /* 8.3 Barrier */
 
 line|z16-[barrier=wall],
-line|z16-[barrier=city_wall],
 line|z16-[barrier=retaining_wall],
 line|z16-[barrier=hedge],
-line|z16-[barrier=fence],
 line|z16-[historic=citywalls],
-{color: @fence;opacity: 1;}
-
+{color: @barriers;opacity: 1;}
+ 
 line|z16[barrier=wall],
-line|z16[barrier=city_wall],
 line|z16[barrier=retaining_wall],
-line|z16[barrier=fence],
 line|z16[barrier=hedge],
 line|z16[historic=citywalls],
 {width: 0.7;dashes: 0.12,0.12;opacity: 0.5;}
 line|z17-[barrier=wall],
-line|z17-[barrier=city_wall],
 line|z17-[barrier=retaining_wall],
-line|z17-[barrier=fence],
 line|z17-[barrier=hedge],
 line|z17-[historic=citywalls],
 {width: 1.0;dashes: 0.25,0.25;opacity:0.7;}
 line|z18-[barrier=wall],
-line|z18-[barrier=city_wall],
 line|z18-[barrier=retaining_wall],
-line|z18-[barrier=fence],
 line|z18-[barrier=hedge],
 line|z18-[historic=citywalls],
 {width: 1.4;dashes: 0.3,0.3;}
+ 
+/* 8.4 Fence */
+ 
+line|z16-[barrier=fence],
+{color: @fencecolor; opacity: 0.5;}
+line|z16[barrier=fence],
+{width: 3.0; dashes: 2.7,2.7; opacity: 0.2;}
+line|z17-[barrier=fence],
+{width: 2.0; dashes: 2.0,2.0; opacity:0.3;}
+line|z18-[barrier=fence],
+{width: 1.0; dashes: 1.5,1.5; opacity: 0.6;}
```

Comment:
Hmm, interesting.
Usually we increase width on higher zooms to make the feature look bigger.
What is the logic to do it vice versa?

Reply:
as per the requirement, 
biker was not able to see the fence on lower zoom level. 
so make it  high width on lower zoom level and make it small width at higher zoom level,
I was thinking in that way.

Reply:
We need to improve visibility on all zoom levels I think so let's keep it consistent with other line-type features rendering - the feature should get bigger the more we zoom in